### DISCUSSION TITLE
Transmission Windows needs killing of Transmission Daemon to work correctly and allow others to access it.

### DISCUSSION THREAD

**Discussion by jan-the-maker** [transmission/transmission#8379] [Support]
### What is the issue?

I realized that I was struggling always with Transmission after restarts. Even when config was set and the UI showed everything correct I was still getting 403 Forbidden when trying to access my WIN machine or no error message but also no entries/transfers. Turns out after a restart of Windows I need to kill at least once the Transmission Daemon and try again to connect from another machine. This is really annoying in the long run. Does the Daemon startup too early and get some incorrect values? Whatever it is can we in worst case just have a GUI icon option to restart the daemon?

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.6

**Comment by Pentaphon** [comment]
Basically you have to go to services.msc and stop the service and restart it. Easiest way to do it. Not sure why you need to stop it to let other people access it though. 

https://www.thewindowsclub.com/open-windows-services

**Comment by jan-the-maker** [comment]
> Basically you have to go to services.msc and stop the service and restart it. Easiest way to do it. Not sure why you need to stop it to let other people access it though.
> 
> https://www.thewindowsclub.com/open-windows-services

I know how to restart a service and that's not the issue. the issue is it shouldn't be necessary. I get different results after a fresh restart and starting transmission (negative aka. 403: Forbidden) and different result when restarting the transmission daemon (positive). 

Anyway I figured out something. I used Powertoys Run function to list the Windows Services and first restarted the Transmission-Daemon (CTRL+R) and it didn't help. multiple times tried. BUT a single STOP process (Enter) did solve it!

Is RESTART not just STOP+START and something different? If yes then maybe that explains something ...

**Comment by DrJW5** [comment]
I have a similar problem.
Often but not always, when I first start Transmission after a Windows restart, torrents are not sending/receiving. 
Killing the process in the task manager and restarting the program solves the issue.
I do not have Transmission auto-starting with Windows, I usually start it manually or the start is triggered by adding a magnet link, so I do not believe it has to do with the program starting "too soon" after Windows.

**Comment by jan-the-maker** [comment]
A "solution" for me is to use powertoys run and kill Transmission Daemon in the process results. super fast. maybe worth looking into. It saved me from going crazy. but good to hear I am not the only one experiencing these problems.
